### ISSUE TITLE
Star 

### ISSUE THREAD

**Issue by f3bandit** [Myself086/Project-Nested#55]

**Comment by johneymute** [comment]
Heeey i got the same massage once i disable stack native return on both tiny toon adventures & adventure island 3 while in mario’s time machine & mario’s missing i do got an error massage saying that a brk code has been detected due lacking of opcodes,and while i did activated allow iligal codes but that didn’t improve things either,while it is true that added generated sram,increasing vram q from 0x1000 to 0x4000 or to 0x8000 does help a little bit but that’s it,they just crash again,
But it’s incredible that once i disabled stack native return,that those games will do show the title screen and intro while otherwise they both show a black screen,so it’s atleast a step forward😁

**Comment by f3bandit** [comment]
> Heeey i got the same massage once i disable stack native return on both tiny toon adventures & adventure island 3 while in mario’s time machine & mario’s missing i do got an error massage saying that a brk code has been detected due lacking of opcodes,and while i did activated allow iligal codes but that didn’t improve things either,while it is true that added generated sram,increasing vram q from 0x1000 to 0x4000 or to 0x8000 does help a little bit but that’s it,they just crash again, But it’s incredible that once i disabled stack native return,that those games will do show the title screen and intro while otherwise they both show a black screen,so it’s atleast a step forward😁

Yeah I got the title screen, intro, and player select screen, but crashes once I get past the dialog to the actual game play.

**Comment by f3bandit** [comment]
So I decided to do an experiment. I downloaded Hakchi2 for the mini consoles and had it apply it's patch for Star Tropics 2 Zoda's revenge. And I got it working some what. I can move around and jump, etc. palette colors still flicker a bit. Maybe this patch can help with nested some how idk!? Settings and the Hakchi patch I used attached below

[Zoda_s_Revenge_StarTropics_II.txt](https://github.com/Myself086/Project-Nested/files/8809880/Zoda_s_Revenge_StarTropics_II.txt)
[d054ffb0-Startropics II - Zoda's Revenge (U) [!] - MMC3 patch.zip](https://github.com/Myself086/Project-Nested/files/8809881/d054ffb0-Startropics.II.-.Zoda.s.Revenge.U.-.MMC3.patch.zip)
!
